pkg/interface/src/views/apps/links/components/LinkDetail.tsx

```diff
@@ -27,12 +27,13 @@ export function LinkDetail(props: LinkDetailProps) {
   return (
     /*  @ts-ignore indio props?? */
     <Row height="100%" width="100%" flexDirection={['column', 'column', 'row']} {...rest}>
-      <LinkBlockItem minWidth="0" minHeight="0" height={['50%', '50%', '100%']} width={['100%', '100%', 'calc(100% - 350px)']} flexGrow={0} border={0} node={node} />
+      <Center flexBasis={['100%', '100%', '75%']} flexShrink={1} overflowY="scroll" >
```

Comment:
The overflow here is good for wider desktop views, but collapses too much on mobile, may need to set an explicit height or adjust the flex styles between this and the sidebar to prevent.

![Screen Shot 2022-04-14 at 3 54 11 PM](https://user-images.githubusercontent.com/5466421/163474474-6d2ebd92-9d96-4a8b-9e9a-2b6083f391cc.png)

Reply:
Tweaked the flex distribution in 4481660842 and c70ca0ad10. Still a bit quirky, but not broken. See gifs below for iphone and ipad screen dimensions, respectively.

It might be worth considering switching from the side-by-side layout here to a more conventional one with comments inline with the document itself, particularly for mobile web: two adjacent vertical scroll boxes is pretty confusing as compared to the old school blogspot-style thing. 

![2022-04-18 15 37 00](https://user-images.githubusercontent.com/57736583/163866725-9190a9be-b2be-4d86-ad59-dd643a93052e.gif)
![2022-04-18 15 38 34](https://user-images.githubusercontent.com/57736583/163866774-b1195353-d213-4cb2-9360-34048aa542e7.gif)